src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -213,23 +213,6 @@ def config(**kwargs):
     redis_kwargs = {k:v for (k,v) in kwargs.items() if k in ['unix_socket_path', 'host', 'port']}
     redis_kwargs['decode_responses'] = True
 
-def init_db():
```

Comment:
>init_db [](http://example.com/codeflow?start=4&length=7)

We could keep the function outside Namespace class, but internally call `Namespace.init_sonic_db_config()`

Reply:
Fixed.